src/balances/balances.controller.spec.ts

```diff
@@ -1,22 +1,68 @@
 import { Test, TestingModule } from '@nestjs/testing';
-import { SafeTransactionModule } from '../services/safe-transaction/safe-transaction.module';
-import { BalancesController } from './balances.controller';
-import { BalancesService } from './balances.service';
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import axios from 'axios';
+import { BalancesModule } from './balances.module';
 
-describe('BalancesController', () => {
-  let controller: BalancesController;
+jest.mock('axios');
+const axiosMock = axios as jest.Mocked<typeof axios>;
+
+describe('AppController (e2e)', () => {
```

Comment:
Nitpick: maybe a more descriptive name could be better? Like `BalancesController` or something similar you like.

Reply:
Changed it here [17d55ed](https://github.com/5afe/safe-client-gateway-nest/pull/15/commits/17d55ede8202804e95b3c194ea9f496d29220a8d)